assets.md

```diff
@@ -134,6 +134,7 @@ Link: https://zapier.com
 This is not really an OpenTelemetry asset as we do not have any credentials or admin access for it.
 
 For support:
+* Check the [EasyCLA status page](https://status.linuxfoundation.org/)
```

Comment:
Is this a superset of https://status.lfx.dev/?

Reply:
I don't know, looks like either would work for our purpose of checking on EasyCLA status